nix/default.nix

```
@@ -99,8 +121,18 @@ in stdenv.mkDerivation rec {
     echo '${androidManifest}' > $out/jni/AndroidManifest.xml
     cd $out && zip -r libwaku.aar *
   '' else ''
-    mkdir -p $out/bin
-    cp -r build/* $out/bin
+    mkdir -p $out/bin $out/include
+    
+    # Copy library files
+    cp build/*.so $out/bin/ 2>/dev/null || true
```

I think we should also need to copy the dynamic macos libs, if any.
```suggestion
    cp build/*.so $out/bin/ 2>/dev/null || true
    cp build/*.dylib $out/bin/ 2>/dev/null || true
```

---

When building libwaku there is no .dylib file, is that something you plan to add?